### PR TITLE
Almabase: Ready for Certification

### PR DESCRIPTION
**Pull request by Almabase-Vaibhav**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a logo.

### CODE REVIEW

File: certified-connectors/Almabase/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": true
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#0942ad",
+    "capabilities": [],
+    "publisher": "Almabase, Inc.",
```
Comment (sriyen-msft) [inline certified-connectors/Almabase/apiProperties.json:32]: @Almabase-Vaibhav Can you please add `stackOwner` as well.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Almabase/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,3634 @@
...
+        "country_short_name": {
+          "description": "The short representation of the country, e.g. US",
+          "example": "US",
+          "readOnly": true,
+          "type": "string",
+          "x-ms-summary": "Country Short Name"
+        },
+        "gpid": {
+          "description": "The place id obtained from Google",
+          "example": "ChIJIQBpAG2ahYAR_6128GcTUEo",
```
[the remark this answers is not in the document]
Comment (Almabase-Vaibhav) [reply certified-connectors/Almabase/apiDefinition.swagger.json]: This is a sample place_id received from Google Autocomplete API. Removed this now.

File: certified-connectors/Almabase/apiProperties.json (hunk 1 above)
Comment (Almabase-Vaibhav) [reply certified-connectors/Almabase/apiProperties.json:32]: Added it. 

When we add stackOwner before running paconn validate, it throws this error. I read this in the limitation of the cli tools.

`{
  "error": {
    "code": "InvalidRequestContent",
    "message": "The request content was invalid and could not be deserialized: 'Could not find member 'stackOwner' on object of type 'ApiPropertiesDefinition'. Path 'properties.stackOwner', line 1, position 655.'."
  }
}
`

**Comment by Almabase-Vaibhav** [comment]
@sriyen-msft  I have made the requested changes.

File: certified-connectors/Almabase/apiProperties.json (hunk 1 above)
Comment (sriyen-msft) [reply certified-connectors/Almabase/apiProperties.json:32]: @diegobernardes Oh, yeah - there is an error in the paconn tool, we are looking into it, thanks for adding manually.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**